v2/cmd/nuclei/main.go

```diff
@@ -104,6 +104,7 @@ on extensive configurability, massive extensibility and ease of use.`)
 		flagSet.FileNormalizedOriginalStringSliceVarP(&options.Workflows, "workflows", "w", []string{}, "list of workflow or workflow directory to run (comma-separated, file)"),
 		flagSet.FileNormalizedOriginalStringSliceVarP(&options.WorkflowURLs, "workflow-url", "wu", []string{}, "list of workflow urls to run (comma-separated, file)"),
 		flagSet.BoolVar(&options.Validate, "validate", false, "validate the passed templates to nuclei"),
+		flagSet.BoolVar(&options.StrictSyntax, "strict-syntax", true, "Force strict syntax check on passed templates to nuclei"),
```

Comment:
What do you think about something like `no-strict-syntax` so that it's not necessary to specify the boolean value?

Reply:
I have no problem with renaming flag names. 
But keep in mind, that current template prior to this PR  **requires** strict syntax. This MR do not change the current behavior of nuclei in handling templates. 

so `strict-syntax=true` or `no-strict-syntax=false` is the current behavior and the user do not need to specify anything (except they want to scan enriched or different templates)

If you want to change the default behavior (to allow custom templates by default) - thats fine for me, but it's a design decision :D

Reply:
I just proposed to rename the flag so that instead of `-strict-syntax=false`, it would be `-no-strict-syntax`, and the default behavior will remain with strict syntax in place. In practice, the flag would become:
```go
flagSet.BoolVar(&options.StrictSyntax, "no-strict-syntax", false, "Disable strict syntax check on templates"),
```